dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/sync/DataSynchronization.java

```diff
@@ -37,22 +37,7 @@
 public abstract class DataSynchronization
 {
     protected boolean syncResult = false;
-    protected int pages;
     protected Clock clock;
     protected int objectsToSynchronize;
     protected SystemInstance instance;
-
-    public abstract SynchronizationResult synchronizeData( final int pageSize );
```

Comment:
Is this abstract class still needed at all?
It seems to me that having an abstract class just with fields reduce a lot the readability of the code.